{{cookiecutter.project_slug}}/runtime.txt

```diff
@@ -1 +1 @@
-python-3.6.2
+python-3.6.3
```

Comment:
`python-3.6.4` is the [latest one](https://devcenter.heroku.com/articles/python-runtimes#supported-python-runtimes).

Reply:
Good catch.  Fixed.